.github/workflows/acceptance-workflow.yml

```diff
@@ -31,27 +31,23 @@ on:
         type: number
         default: 3
     secrets:
-      operator_key:
-        description: 'The ED25519, ECDSA, or DER encoded private key of the operator'
-        required: false
       CODECOV_TOKEN:
         description: 'Codecov upload token'
         required: true
 
-
-env:
-  OPERATOR_ID_MAIN: ${{ inputs.operator_id }}
-
 jobs:
   acceptance-workflow:
     runs-on: smart-contracts-linux-large
     timeout-minutes: 50
     permissions:
       contents: read
       checks: write
-    env:
-      OPERATOR_ID_MAIN: ${{ inputs.operator_id }}
     steps:
+      - name: Set env variables
```

Comment:
Is there a reason to remove the `env` statement and use this `sh` snippet?

Reply:
Yes, it's an optional input variable which is wrongly set always even when its not provided, causing further difficulties in the configurations, as seen in the deleted code that was trying to "workaround" this issue from the workflow

Reply:
Whereas, doing:
```
    env:
       OPERATOR_ID_MAIN: ${{ inputs.operator_id }}
```

would override the one from the `.env` file even if the input is empty, which is causing the difficulties in the code we are trying to fix.